.env.example

```diff
@@ -6,7 +6,7 @@
 APP_URL=
 # A unique key to identify the app.
 # See for more detail: https://developer.atlassian.com/cloud/jira/platform/connect-app-descriptor/
-APP_KEY=com.figma.jira-addon-dev
```

Comment:
to align this with the value we use on the frontend

Reply:
**praise:** Nice catch, Aditi!